### PR TITLE
support DiT ImageNet dataset training

### PR DESCRIPTION
**Pull request by zhtmike**
# What does this PR do?

- Support DiT training with ImageNet dataset format, using `--imagenet_format` flag
- Fix the loss scale print in callback
- clean code

Fixes # (issue)

Adds # (feature)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

File: examples/dit/train.py [hunk 1]
```diff
@@ -288,7 +306,7 @@ def main(args):
             log_interval=args.callback_size,
             start_epoch=start_epoch,
             model_name="DiT",
-            record_lr=False,  # TODO: check LR retrival for new MS on 910b
+            record_lr=True,
```
Comment (hadipash) [inline examples/dit/train.py:309]: Does it work now on 910B?

**Review by hadipash — APPROVED** [review]

File: examples/dit/train.py (hunk 1 above)
Comment (zhtmike) [reply examples/dit/train.py:309]: yes

**Comment by wtomin** [comment]
It looks great! Can you also provide a bash script to launch a four-card training, because I notice DiT torch repo is using a global batch size 256 for ImageNet training.

**Comment by zhtmike** [comment]
> It looks great! Can you also provide a bash script to launch a four-card training, because I notice DiT torch repo is using a global batch size 256 for ImageNet training.
> 
> Besides, please revise the readme file for the ImageNet training. I think users are more interested in this rather than a toy dataset training.

Done. Add a 4P training script and update the README

**Review by wtomin — APPROVED** [review]
